### PR TITLE
Yeet custom config in favor of NHLib

### PR DESCRIPTION
**Pull request by ah-OOG-ah**
We only need one cursed extension of forge's config in this pack.
This shouldn't change any behavior - however, it completely ignores the prior configs and replaces them with two, `config/ExtraTrees.cfg` and `config/ExtraBees.cfg`.

The module enable/disables have been completely removed as we do not support that, and various config other files/options were removed as they were either unused in code or empty.

Some config options now work, as previously they were disconnected for no apparent reason.

### CODE REVIEW

**Comment by wlhlm** [comment]
I'd merge this after 2.7 as it doesn't fix any known bugs (as far as I'm aware) and can potentially introduce new ones.

File: src/main/java/binnie/extrabees/config/ConfigurationMachines.java [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-    @PropPercentage
-    public static int isolatorConsumptionChance = 30;
-
-    @ConfigProperty(
-            key = "geneticErrorModifier",
-            comment = { "Modifier that changes how severe splicer/inoculator erros are.",
-                    "0 would result in no errors, even for awful serums.",
-                    "2.0 would result in twice as severe errors" })
-    @PropDouble
-    public static double geneticErrorModifier = 1.0;
```
Comment (serenibyss) [inline src/main/java/binnie/extrabees/config/ConfigurationMachines.java:23]: Was this config option unused?

File: src/main/java/binnie/extrabees/config/EBConfigGUIFactory.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package binnie.extrabees.config;
+
+import net.minecraft.client.gui.GuiScreen;
+
+import com.gtnewhorizon.gtnhlib.config.SimpleGuiFactory;
+
+public class EBConfigGUIFactory implements SimpleGuiFactory {
```
Comment (serenibyss) [inline src/main/java/binnie/extrabees/config/EBConfigGUIFactory.java:7]: Are these factories (two of them) doing anything special? They are no longer needed for gtnhlib configs if the standard gui works for what you want

**Review by serenibyss — COMMENTED** [review]
Overall looks great, just a couple small comments, also mentioning that we will need to update the config files in the modpack repo so we don't forget

**Review by serenibyss — APPROVED** [review]

**Pull request merged**